### PR TITLE
PlayerNetworkStatusPacket

### PR DESCRIPTION
**Pull request by CrosRoad95**
I don't know how to test it on local server

### CODE REVIEW

File: SlipeServer.Packets/Definitions/Player/PlayerNetworkStatusPacket.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        {
+        }
+
+        public override void Read(byte[] bytes)
+        {
+            PacketReader reader = new PacketReader(bytes);
+            switch(reader.GetByte())
+            {
+                case 0:
+                    this.Type = PlayerNetworkStatusType.InterruptionBegan;
```
Comment (NanoBob) [inline SlipeServer.Packets/Definitions/Player/PlayerNetworkStatusPacket.cs:26]: You don't need this switch case, you can just cast the byte to `PlayerNetworkStatusType`

**Review by NanoBob — CHANGES_REQUESTED** [review]
Do make sure you test it, either ask someone else to connect to your server or use a VM for testing

**Comment by CrosRoad95** [comment]
To test it on local server, you need to attach debugger to gta_sa.exe and pause process, you will notice messages appearing on server. When unpaused you will see how long client was hang
image

**Review by NanoBob — APPROVED** [review]

**Comment by th1302** [comment]
Can you relpy my isssues

**Pull request merged**